zerver/lib/bulk_create.py

```diff
@@ -16,6 +18,7 @@ def bulk_create_realms(realm_list):
     Realm.objects.bulk_create(realms_to_create)
 
 def bulk_create_users(realms, users_raw, bot_type=None):
+    # type: (Dict[Any, Realm], Set[Tuple[Any, Any, Any, bool]], bool) -> None
```

Comment:
Also, I think we know the types for the arguments in users_raw, aren't they all text_type as well?